CHANGELOG.md

```diff
@@ -4,6 +4,62 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.12.0 (2023-09-20)
+
+This release contains performance improvements for compressed hypertables
+and continuous aggregates and bug fixes since the 2.11.2 release.
+We recommend that you upgrade at the next available opportunity.
+
+This release moves all internal functions from the _timescaleb_internal
+schema into the _timescaledb_functions schema. This separates code from
+internal data objects and improves security by allowing more restrictive
+permissions for the code schema. If you are calling any of those internal
+functions you should adjust your code as soon as possible. This version
+also includes a compatibility layer that allows calling them in the old
+location but that layer will be removed in 2.14.0.
+
```

Comment:
```suggestion

**PostgreSQL 12 support removal announcement**
Following the deprecation announcement for PostgreSQL 12 in TimescaleDB 2.10,
PostgreSQL 12 is not supported starting with TimescaleDB 2.12.
Please upgrade your PostgreSQL server to a newer PostgreSQL major version to 
be able to upgrade to TimescaleDB 2.12.
Currently supported PostgreSQL major versions: 13, 14, 15
PostgreSQL 16 support will be added with a following TimescaleDB release.
```

Reply:
What do you think @svenklemm? Maybe too wordy? I thought of adding the last 2 lines to be safe

Reply:
Maybe cut out the middle sentence but other than that looks good to me